keyboards/namecard2x4/rev2/config.h

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2018 takashiski
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+
+#undef MATRIX_ROW_PINS
+#undef MATRIX_COL_PINS
+
+#define MATRIX_ROW_PINS { B2,B6 }
+#define MATRIX_COL_PINS { D7,E6,B4,B5 }
+#undef DIODE_DIRECTION
```

Comment:
Shouldn't need to undef this.

Just move the `DIODE_DIRECTION` setting to the rev1's config.h instead.

Reply:
@drashna `namecard2x4/rev1/config.h` doesn't have any settings on its own, but inherits from `namecard2x4/config.h` currently.

[keyboards/namecard2x4/rev1/config.h](https://github.com/qmk/qmk_firmware/blob/7c378205a0af6c3f6cef26b2ffcf5da503757f3b/keyboards/namecard2x4/rev1/config.h) vs. [keyboards/namecard2x4/config.h#L48-L49](https://github.com/qmk/qmk_firmware/blob/7c378205a0af6c3f6cef26b2ffcf5da503757f3b/keyboards/namecard2x4/config.h#L48-L49)

Reply:
Sorry, yeah, that's what I meant, and I wasn't too clear.

It's better to not undef here, and to just move the definition from the main config.h file, to the rev1 config.h.

Reply:
Thank you for reviewing.
First idea is same your idea. but it has multiple definition error in ` make namecard2x4`.

I want that build command is used only `make namecard2x4/rev1` or `make namecard2x4/rev2`, but travisci try `make namecard2x4`, too. 
Latest commit is fixed it. 

Do you have better idea?

Reply:
@takashiski Travis CI tries to build `make namecard2x4` because there is a `rules.mk` file in `keyboards/namecard2x4`. If you refactor the keyboard so you only have `keyboards/namecard2x4/rev1/rules.mk` and `keyboards/namecard2x4/rev2/rules.mk`, that should stop.

Reply:
Or you can use `DEFAULT_FOLDER = namecard2x4/rev1`